gnomad/resources/grch38/gnomad.py

```diff
@@ -221,7 +221,10 @@
 """
 
 POPS_STORED_AS_SUBPOPS = TGP_POPS + HGDP_POPS
-POPS_TO_REMOVE_FOR_POPMAX = {"asj", "fin", "oth", "ami", "mid", "remaining"}
+POPS_TO_REMOVE_FOR_POPMAX = {
+    "v3": {"asj", "fin", "mid", "oth", "ami", "remaining"},
+    "v4": {"asj", "fin", "oth", "ami", "remaining"},
```

Comment:
either way this is fine, just wondering does v3 have "remaining" and does v4 have "oth"? i don't remember what steps the name changes went in

Reply:
Yeah this is a bit annoying, v4 genomes uses remaining but v3 genomes used oth but the code needs both for backwards compatibility since the v4 genomes uses the v3 set since we still have fewer than 1000 mid samples in it....this is why we need a resources overhaul.

Reply:
I kept oth in v4 as a safety measure here because of the joint_frequency work, thats also why ami is in v4 even though v4 exomes do not have ami samples